terraform/vars/skylight/main.tf

```diff
@@ -24,3 +24,13 @@ module "network" {
   source = "../../modules/network"
 }
 
+module "fhir" {
+  source         = "../../modules/fhir"
+  region         = "us-west1"
+  time_zone      = "UTC"
+  dataset_name   = "PHDI_DATASET-${time_static.current_date.year}${time_static.current_date.month}${time_static.current_date.day}"
+  fhirstore_name = "PHDI_FHIRSTORE-${time_static.current_date.year}${time_static.current_date.month}${time_static.current_date.day}"
```

Comment:
I _think_ there is not the same uniqueness restriction on fhir store names because they are all within datasets that must have unique names.

Reply:
It must be unique within the dataset though.  In the docs it says:  "Enter a name of your choice that's unique in your dataset. If the name is not unique, the FHIR store creation fails."

Reply:
Oh right that makes sense. Let's leave this how it is then.